Packs/DevSecOps/IncidentFields/DevSecOps_After_Commit_ID.json

```
@@ -0,0 +1,30 @@
+{
+    "id": "incident_aftercommitid",
```

```suggestion
    "id": "incident_devsecopsaftercommitid",
``` 
Same in all incident fields/types indicator fields/types